### PR TITLE
processing descriptions for site locations

### PR DESCRIPTION
**Pull request by jread-usgs**
more for #249

### CODE REVIEW

File: 4_data_release/in/metadata_site_points.yaml [hunk 1]
```diff
@@ -10,7 +10,11 @@ title: Spatial site locations
 process-description: >- 
   A points shapefile was created containing locations of the USGS gaging stations included in this data release. 
   Spatial tools in the R programming language were used to perform the operations of creating the spatial datasets and modifying attributes. 
-  Station locations were accessed from the USGS National Water Information System's "site service": https://waterservices.usgs.gov/rest/Site-Service.html
+  Station locations were accessed from the USGS National Water Information System's "site service": 
+  https://waterservices.usgs.gov/rest/Site-Service.html. The R dataRetrieval package was used to access site service metadata 
+  (function readNWISsite). Sites had one of two datums, NAD83 (epsg code 4269) 
+  or WGS84 (epsg code 4326) and were transformed to a single datum (epsg code 4326) using the rgdal package. The locations of each site 
+  were combined into a single output shapefile. Details for package versions used are fount in the build environment details in <native>.
```
Comment (aappling-usgs) [inline 4_data_release/in/metadata_site_points.yaml]: ScienceBase renders metadata without tags such as `<native>`, so I've been referring to sections by their longer names - in this case "Native Data Set Environment". I suppose there are pros and cons to each...what do you think?

File: 4_data_release/in/metadata_parent.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ themekeywords: ["water","streams","rivers","metabolism","primary productivity","
 usage-rules: >-
   None, but see Distribution Liability below. 
   Users are advised to read the associated publication thoroughly to understand appropriate use and data limitations.
-doi: http://dx.doi.org/10.5066/REPLACE.w.REAL
+doi: https://doi.org/10.5066/F70864KX
```
Comment (aappling-usgs) [inline 4_data_release/in/metadata_parent.yaml]: :+1:

File: 4_data_release/in/metadata_site_points.yaml (hunk 1 above)
Comment (jread-usgs) [reply 4_data_release/in/metadata_site_points.yaml]: yeah, I was thinking the `native` tag isn't very useful too - so I will switch it.

**Comment by aappling-usgs** [comment]
also completes #244 for site points and catchments, and now the other 7 metadata files just need to be rebuilt & reposted and they'll have the new DOI too.

**Pull request merged**